src/Modifier/Common/AttributeModifier.php

```diff
@@ -94,7 +98,17 @@ class AttributeModifier extends Modifier
 
     private Connection $database;
 
-    private string $activeSnippet;
+    /**
+     * @var BaseModel
+     */
+    private $model = null;
+
+    /**
+     * @var string
+     */
+    private $activeSnippet;
```

Comment:
Type hint is missing.

```suggestion
    private ?string $activeSnippet = null;
```